Figure_02_01.py

```diff
@@ -1,16 +1,16 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
-w = 0.2
+w = 0.2 #rad/s
 s = w*1j
 G = 5*np.exp(-2*s)/(10*s + 1)
 
 t = np.linspace(0, 100, 200)
-GM = np.abs(G)
-PM = np.angle(G)
+GN = np.abs(G)      #Gain at w = 0.2 rad/s
```

Comment:
The GM for this transfer function is 1.7 at 0.8 rad/s, and PM is 45.4 at 0.5 rad/s.  That which is calculated on line [9] is the System Gain at 0.2 rad/s (in which case SG would probably have been a better acronym).